tmt/steps/provision/local.py

```diff
@@ -136,7 +148,30 @@ def execute(
         )
 
     def install_scripts(self, scripts: Sequence[tmt.steps.scripts.Script]) -> None:
-        self.debug("No installation of tmt scripts is needed on localhost.")
+        """Install scripts required by tmt"""
+
+        # Make sure scripts directory exists
+        command = Command("mkdir", "-p", f"{self.scripts_path}")
+
+        if not self.facts.is_superuser:
+            command = Command("sudo") + command
+
+        self.execute(command, silent=True)
```

Comment:
Implementation wise, I think it should follow https://github.com/teemtee/tmt/pull/3983

Reply:
I everybody agrees I will send an update to follow the same logic

Reply:
@mcasquer I believe this is what we agreed upon yes